### PR TITLE
[Feat]: badges

### PR DESCRIPTION
**Pull request by aryan-13**
-Badges added to the documentation

badges-1

### CODE REVIEW

**Review by raghav2110 — COMMENTED** [review]
Looks code for merge

File: components/badges.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	display: flex;
+	color: white;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	font-size: 0.75em;
+	box-sizing: border-box;
+}
+.badge-online {
+	background-color: #00b300;
```
Comment (Hamdrive) [inline components/badges.css:17]: Use color variable here instead of hexcode, same for dnd

File: components/badges.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	padding: 2px;
+	border-radius: 50%;
+	display: flex;
+	color: white;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	background-color: var(--color-primary);
+}
+.badge-box .avatar-xs ~ .img-badge {
```
Comment (Hamdrive) [inline components/badges.css:35]: What is this character here doing `~` ?

**Review by Hamdrive — COMMENTED** [review]
Overall looks good.

File: components/badges.css (hunk 2 above)
Comment (aryan-13) [reply components/badges.css:35]: It selects the siblings' element.
Here is the article I read: https://www.w3schools.com/css/css_combinators.asp

File: components/badges.css (hunk 1 above)
Comment (aryan-13) [reply components/badges.css:17]: These colors are not supposed to change, so that's why I used the actual hexcode

**Pull request merged**